### DISCUSSION TITLE
Wide-angle camera on Samsung Galaxy S11 and others

### DISCUSSION THREAD

**Discussion by AlexPahar** [mebjas/html5-qrcode#710] [Q&A]
There was a problem. There is a problem on some smartphones. The wide-angle camera turns on, which causes problems with scanning and display, while everything works fine on other smartphones.
Please tell me, has anyone experienced something similar? If yes, please tell me how you solved this problem.

**Comment by mebjas** [comment]


**Reply by mebjas** [reply]
I plan to make it easier to switch the cameras (without stopping camera)
